### PR TITLE
Google

### LINKED ISSUE #99: Google
- [x] add google sign in
- [x] change the site to be a more mobile freindly site

### PR DESCRIPTION
**Pull request by MortarDefender**
- added google sign-in button
- changed HTML and CSS files for a more mobile-friendly experience

- fix: #99

### CODE REVIEW

**Comment by kobihk** [comment]
@MortarDefender please fix the conflicts

**Comment by kobihk** [comment]
@MortarDefender please ask for CR if you want to merge it before the demo

File: events/static/meetings/create_meeting.css [hunk 1]
```diff
@@ -1,7 +1,8 @@
 body {
     background: url("https://cdn.wallpapersafari.com/1/72/14W9Zo.jpg") no-repeat;
+    background-attachment: fixed;
     background-size: cover;
-    background-repeat: repeat;
+    /* background-repeat: repeat; */
```
Comment (YaelD) [inline events/static/meetings/create_meeting.css]: Please remove this line

File: lets_meet/.env [hunk 2]
```diff
@@ -4,3 +4,5 @@ EMAIL_PORT='587'
 EMAIL_USER='letsmeetsquad@gmail.com'
 EMAIL_PASSWORD='irvrjnyzfqcyxntj'
 EMAIL_TLS=True
+AUTH_ID="370195302414-q0sob5lr972rrrkv55qultt61oard9nu.apps.googleusercontent.com"
+AUTH_SECRET="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"
```
Comment (YaelD) [inline lets_meet/.env]: Please add blank line in the end of the file

File: users/static/base.css [hunk 3]
```diff
@@ -34,7 +34,7 @@
   cursor: pointer;
 
   opacity: 0; /* hide this */
-  z-index: 2; /* and place it over the hamburger */
+  z-index: 8; /* and place it over the hamburger */
```
Comment (YaelD) [inline users/static/base.css]: Can you please remove this comment if it is no longer needed?

File: users/static/login/login.css [hunk 4]
```diff
@@ -221,3 +221,44 @@ body {
...
+			font-size: xx-large;
+	}
+
+	.container {
+			height: 130%;
+			max-width: 100%;
+			width: 100%;
+			top: -50%;
+	}
+}
```
Comment (YaelD) [inline users/static/login/login.css]: Please fix the indentations here

File: users/static/site_notification/notification.js [hunk 5]
```diff
@@ -3,6 +3,11 @@ let get_notification = "";
 let seen_notification = "";
 
 
+function move(url) {
+	console.log(url);
```
Comment (YaelD) [inline users/static/site_notification/notification.js]: Please remove this line

File: users/static/update_profile/update_form.css [hunk 6]
```diff
@@ -192,50 +192,128 @@ footer .fp a:hover {
...
+  }
+
+  label {
+    font-size: 1.5rem;
+  }
+
+  #input-title, #location, #id_date_time_end, #id_date_time_start {
+    font-size: 1.3rem;
+  }
+}
```
Comment (YaelD) [inline users/static/update_profile/update_form.css]: Please add a blank line in the end of the file

**Review by YaelD — CHANGES_REQUESTED** [review]

File: lets_meet/.env (hunk 2 above)
Comment (MortarDefender) [reply lets_meet/.env]: done

File: users/static/base.css (hunk 3 above)
Comment (MortarDefender) [reply users/static/base.css]: done

File: users/static/login/login.css (hunk 4 above)
Comment (MortarDefender) [reply users/static/login/login.css]: done

File: users/static/update_profile/update_form.css (hunk 6 above)
Comment (MortarDefender) [reply users/static/update_profile/update_form.css]: done

**Comment by MortarDefender** [comment]
image
image

**Review by RonHachmon — APPROVED** [review]
Looks great 👍

**Review by YaelD — APPROVED** [review]
LGTM 👍

**Review by kobihk — APPROVED** [review]

**Pull request merged**